code/data_structures/linked_list/linked_lists_operations/insertionInALinkedList.cpp

```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+using namespace std;
+
+// Eveerytime you work with Append or Push function, there is a corner case. That when a NULL node is passed you have to return a node that points to the given value. There are two ways you can do, if you receive the double pointer as an argument or if you are returning node.
+
+struct node{
```

Comment:
According to our style guide, node should be capitalized. Also, instead of making a struct, which leads to a lot of pointer manipulation, consider making a linked list class with member functions, and generality. (Just a recommendation). Why use structs in C++ when C suffices enough?

Reply:
Is it okay if I made the required changes for now & then try doing it by making a Linked List class later?